gravitee-apim-gateway/gravitee-apim-gateway-standalone/gravitee-apim-gateway-standalone-distribution/src/main/resources/config/gravitee.yml

```diff
@@ -154,6 +154,41 @@ ratelimit:
   type: mongodb
   mongodb:
     uri: mongodb://${ds.mongodb.host}:${ds.mongodb.port}/${ds.mongodb.dbname}
+#  redis:
+#    host: localhost
+#    port: 6379
+#    password:
+#    # Redis Sentinel settings
+#    sentinel:
+#      master: redis-master
+#      nodes:
+#        - host: sentinel1
+#          port: 26379
+#        - host: sentinel2
+#          port: 26379
+#    # SSL settings
+#    ssl: true
```

Comment:
Usually, we put the default value in the gravitee.yml for such configuration (ssl, trustAll, ...)

Reply:
Ok, I'll update the value to match with the default ones.